cli/cmd/lql.go

```diff
@@ -44,6 +44,7 @@ var (
 		Range         string
 		Start         string
 		URL           string
+		Language      string
```

Comment:
Fundamentally I don't think we need to add `--language` as a flag at all given that it's part of the query markup (for the same reason that queryId and queryText are not flags...

Reply:
Ah.  I think I see what's going on.  This is for us to be able to help pre-populate if using the editor.  I think it might be cleaner to simply use a survey.Select with `LQL` and `Rego` options, only if and when we're in the `inputQueryFromEditor` flow...

I think adding a flag is confusing when creating query from file, url, etc. when this will already have the `queryLanguage` specified.

Reply:
We are adding this option to decide which query template to provide in EDITOR mode - Rego query is different from LQL query.

Reply:
Can you give me an example of how a survey is defined? @hazedav

Reply:
Sure thing bud.  We use `survey.Select` a bunch in go-sdk.  Here is one such example:
https://github.com/lacework/go-sdk/blob/06b930715167c526ce9f8301d9a0cae23c8e9bdc/cli/cmd/container_registry.go#L160

Basically before opening the editor we'll prompt the user to select their desired language.